docs/more-pushmsg.md

```diff
@@ -83,3 +83,36 @@ switch ($code) {
         echo "<strong>Hmm Something Went Wrong or HTTP Status Code is Missing</strong>";
 }
 ```
+
+### Node.js - Using Axios
```

Comment:
I'd prefer to name this just JavaScript, there is nothing which prevents this from running in a non nodejs environment.
```suggestion
### JavaScript
```